stories/main.scss

```diff
@@ -0,0 +1,7 @@
+.docs-story {
```

Comment:
Well, if you don't mind, it would be nice to put here a comment to explain the reason (the **why**) for this css trick (possibly with a link to the storybook issue if it exists).